packages/i18next/src/lib/functions.ts

```diff
@@ -1,16 +1,16 @@
 import { container } from '@sapphire/pieces';
 import { isObject, NonNullObject } from '@sapphire/utilities';
-import { Guild, Message, MessageOptions } from 'discord.js';
+import { Guild, Message } from 'discord.js';
 import type { StringMap, TFunctionKeys, TFunctionResult, TOptions } from 'i18next';
-import type { DiscordChannel, InternationalizationContext, TextBasedDiscordChannel } from './types';
-
-export type ChannelTarget = Message | DiscordChannel;
-export type Target = ChannelTarget | Guild;
+import type { ChannelTarget, InternationalizationContext, LocalizedMessageOptions, Target, TextBasedDiscordChannel } from './types';
 
 /**
- * Retrieves the language name for a specific target, using {@link InternationalizationHandler.fetchLanguage}, and if it
- * returns a nullish value, then it falls back to {@link Guild.preferredLocale}, then
- * {@link InternationalizationOptions.defaultName} if no guild was provided, and finally 'en-US' if none was set.
+ * Retrieves the language name for a specific target, using {@link InternationalizationHandler.fetchLanguage}.
+ * If {@link InternationalizationHandler.fetchLanguage} is not defined, or returns a nullish value,
```

Comment:
I'd remove this comma here, feels unnatural to read
`is not defined, or returns` => `is not defined or returns`